src/please.go

```diff
@@ -251,6 +252,14 @@ var opts struct {
 		} `command:"outputs" description:"Exports outputs of a set of targets"`
 	} `command:"export" subcommands-optional:"true" description:"Exports a set of targets and files from the repo."`
 
+	Format struct {
+		Quiet bool `long:"quiet" short:"q" description:"Don't print corrections to stdout"`
```

Comment:
I guess it's obvious that this returns a non-zero code that you can use for just doing linting?